sdk/resources/Azure.Management.Resources/tests/Azure.Management.Resources.Tests.csproj

```diff
@@ -1,5 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);CA1812;SA1633;SA1649</NoWarn>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" />
+  </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\src\Azure.Management.Resources.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="ScenarioTests\*.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
```

Comment:
please remove it

Reply:
```
<ItemGroup>
    <None Update="ScenarioTests\*.json">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
  </ItemGroup>
```
This block is need to copy the test required json template file.